src/pudl/metadata/resources/eia.py

```diff
@@ -6,6 +6,54 @@
 
 AGG_FREQS = ["yearly", "monthly"]
 
+HARVESTING_DETAIL_TEXT = """EIA reports many attributes in many different tables across
+EIA-860 and EIA-923. In order to compile tidy, well-normalized database tables, PUDL
+collects all instances of these values and and chooses a canonical value. By default,
+PUDL chooses the most consistently reported value of a given attribute as long as it
+is at least 70% of the given instances reported. If an attribute was reported
+inconsistently across the original EIA tables, then it will show up as a
+null value."""
+
+
+def canonical_harvested_details(entities: str, is_static: bool) -> str:
+    """Generate additional details text for one of the eight core harvested tables.
+
+    We have one core harvested table for each combination of (plants, utilities,
+    boilers, generators) X (static cols, annual cols). This text helps users
+    cross reference where the canonical values for each type of entity comes
+    from, and why it may differ from a value they find in a raw source.
+
+    Args:
+        entities: string containing the plural of an entity type; e.g., "plants"
+        is_static: True if the table this text is destined for contains the static cols
+            for the entity, False otherwise. Static cols are stored in tables with a name
+            like "core_eia__entity_X", and annual cols are stored in tables with a name
+            like "core_eia860__scd_X".
+    """
+    return f"""This is one of two tables where canonical
+values for {entities} are set. It contains values which are expected to {"remain fixed" if is_static else "vary slowly"}, while
+:ref:`core_eia{"860" if is_static else ""}__{"scd" if is_static else "entity"}_{entities}` contains those {"which may vary from year to year" if is_static else "expected to remain fixed"}.
+{HARVESTING_DETAIL_TEXT}
+All tables downstream of this one inherit the canonical values established here."""
+
+
+def inherits_harvested_values_details(entities: str) -> str:
+    """Generate additional details text for a table which inherits harvested values from one of the eight core harvested tables.
+
+    A table inherits harvested values from one of the eight core harvested tables
+    if it is downstream of one or more tables ``core_eia__entity_{plants|utilities|boilers|generators}``
+    or ``core_eia860__scd_{plants|utilities|boilers|generators}`` and includes one or
+    more columns from the static or annual column lists in :data:`pudl.metadata.resources.ENTITIES`.
```

Comment:
We're only labeling tables that inherit 3 or more columns, but technically tables that inherit 1 or 2 columns count for this paragraph. is that confusing?